stl/CMakeLists.txt

```diff
@@ -507,6 +507,9 @@ function(add_stl_dlls D_SUFFIX REL_OR_DBG)
 
     add_library(msvcp${D_SUFFIX} SHARED)
     target_link_libraries(msvcp${D_SUFFIX} PRIVATE msvcp${D_SUFFIX}_eha_objects msvcp${D_SUFFIX}_objects msvcp${D_SUFFIX}_init_objects "${TOOLSET_LIB}/vcruntime${D_SUFFIX}.lib" "${TOOLSET_LIB}/msvcrt${D_SUFFIX}.lib" "ucrt${D_SUFFIX}.lib")
+    if(VCLIBS_TARGET_ARCHITECTURE MATCHES "^arm64(ec)?$")

```

Comment:
Are you sure ARM64 non EC needs that?
It somehow linked and built tests before the change.

Reply:
I'm not sure. `msvcp.settings.targets` seems to do this, but I don't understand why.

https://github.com/microsoft/STL/blob/da7307d68dbf39a805372e91a740393cbf06595c/stl/msbuild/stl_base/msvcp.settings.targets#L74